lib/learn_to_rank/ranker.rb

```diff
@@ -60,7 +60,7 @@ def tensorflow_serving_ip
     end
 
     def ranker_error(response)
-      response.nil? || response.code != 200
+      response.body.nil? || response.body.empty? || response.code != 200
```

Comment:
I think `response.body.nil? || response.body.empty?` could be replaced with `response.body.blank?`

```
Loading development environment (Rails 5.2.4)
[1] pry(main)> nil.blank?
=> true
[2] pry(main)> " ".blank?
=> true
[3] pry(main)> "".blank?
=> true
```

Reply:
I'd thought that as search-api is Sinatra, it wouldn't have `blank?`.  Turns out we have ActiveSupport added so I can use it.  Thanks

Reply:
Ah yes, actually hadn't realised that. Great!